tensorflow_datasets/image/cityscapes.py

```diff
@@ -0,0 +1,185 @@
+# pylint: disable=bad-indentation
```

Comment:
any reason why ? ;-)

Reply:
Sorry! My global pylint config uses 4 spaces which causes a lot of false positives in Google style python code! Will remove!